bridgetown-core/lib/bridgetown-core/liquid_renderer/file_system.rb

```diff
@@ -24,13 +37,35 @@ def read_template_file(template_path)
         # Restore pristine state
         self.root = load_paths
 
-        found_paths.compact!
+        found_paths.compact
+      end
 
-        raise Liquid::FileSystemError, "No such template '#{template_path}'" if found_paths.empty?
+      # rubocop:disable Metrics/AbcSize
+      def parse_liquid_component(cache_key, filename)
+        file_content = ""
 
-        # Last path in the list wins
-        ::File.read(found_paths.last)
+        # Strip YAML header
+        if Utils.has_yaml_header?(filename)
+          begin
+            content = ::File.read(filename)
+            file_content = $POSTMATCH if content =~ Document::YAML_FRONT_MATTER_REGEXP
+          rescue Psych::SyntaxError => e
```

Comment:
I just realized we don't need this since we're not actually parsing the YAML, just stripping it out :)